CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # This cmake build is for Windows 64-bit only.
 #
 # Prerequisites:
-#     You must have Visual Studio 2013 Update 4 installed. Start the Developer Command Prompt window that is a part of Visual Studio installation.
+#     You must have at least Visual Studio 2015. Start the Developer Command Prompt window that is a part of Visual Studio installation.
```

Comment:
Add preferably with the latest update on top of it that address compiler bug fixes.